Immense.RemoteControl.Server/wwwroot/css/remote-control.css

```diff
@@ -240,6 +241,10 @@ button {
     display: none !important;
 }
 
+.ms-1 {
```

Comment:
minor: what does `.ms` stand for? The bootstrap way would be `ml` for "margin-left"

Reply:
In [Bootstrap 5](https://getbootstrap.com/docs/5.0/utilities/spacing/#notation), they changed it to `ms-#` and `me-#` (start and end).

Reply:
did not know that! thanks